File: backendOrderly/src/main/java/com/yebur/backendorderly/overview/OrderWithOrderDetailResponse.java
package com.yebur.backendorderly.overview;

import java.time.LocalDateTime;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class OrderWithOrderDetailResponse {
    private String overviewId;
    private Long orderId;
    private String tableName;
    private List<OrderDetailSummary> details;
    private LocalDateTime datetime;
}

@Data
@AllArgsConstructor
@NoArgsConstructor
class OrderDetailSummary {
    private Long id;
    private String productName;
    private String comment;
    private int amount;
    private String status;
    private String destination;
}

File: frontend/OrderlyTablet/app/src/main/java/com/example/orderlytablet/response/OrderWithOrderDetailResponse.kt
package com.example.orderlytablet.response

data class OrderWithOrderDetailResponse(
    val id: Long,
    val tableName: String,
    val details: List<OrderDetailSummary>,
    val datetime: String
)

data class OrderDetailSummary(
    val id: Long,
    val productName: String,
    val comment: String,
    val amount: Int,
    val status: String
)